pal_event_server/lib/src/allowlist.dart

```diff
@@ -14,129 +14,142 @@
 
 // @dart=2.9
 
-import 'url_matcher.dart';
+import 'package:pal_event_server/src/loggers/pal_event_helper.dart';
+import 'package:taqo_common/model/event.dart';
+import 'dart:convert';
+import 'package:crypto/crypto.dart';
+import 'package:logging/logging.dart';
+
+const APPS_USED_RULE_TYPE = 'apps_used';
+const APP_CONTENT_RULE_TYPE = 'app_content';
+
+final _logger = Logger('AllowListLogger');
+
+class AllowListRule {
+  String _type;
+  RegExp _expression;
+  String _appForContentRule;
+
+  AllowListRule(Map<String, String> map) {
```

Comment:
That is a good idea long term to switched to named parameters. 

I used the map for now because I think we are definitely going to want to change the format of allowlist rules.

Reply:
Is this OK to you?

Reply:
Sure.